_build/utils/propagate.py

```diff
@@ -491,7 +492,7 @@ def compile_ud(tmp, gum_target, reddit=False):
 
 		processed_lines = "\n".join(processed_lines) + "\n"
 		# Serialize entity tagged dependencies for debugging
-		with io.open(tmp + "entidep" + os.sep + docname + ".conllu",'w',encoding="utf8", newline="\n") as f:
+		with io.open(tmp + os.sep + "dep" + os.sep + "tmp" + os.sep + docname + ".conllu",'w',encoding="utf8", newline="\n") as f:
```

Comment:
@nitinvwaran I think this is not meant to be the directory used by pepper for input, it's just a serialization of the conll data with extra features in the FEATS column called ent_head and infstat (see comment on line 473)